### PR TITLE
Add option to color by hydrostatic for fit grains

### PR DESCRIPTION
**Pull request by psavery**
This adds an option to color by hydrostatic stress/strain:

hydrostatic_stress

Fixes: #673

### CODE REVIEW

File: hexrd/ui/indexing/fit_grains_results_dialog.py [hunk 1]
```diff
@@ -60,14 +60,17 @@ def __init__(self, data, material=None, parent=None):
...
-        # Add column for equivalent strain
+        # Add columns for equivalent strain and hydrostatic strain
         eqv_strain = np.zeros(self.num_grains)
+        hydrostatic_strain = np.zeros(self.num_grains)
         for i, grain in enumerate(self.data):
             epsilon = vecMVToSymm(grain[15:21], scale=False)
             deviator = epsilon - (1/3) * np.trace(epsilon) * np.identity(3)
             eqv_strain[i] = 2 * np.sqrt(np.sum(deviator**2)) / 3
-        # Reshape so we can hstack it
+            hydrostatic_strain[i] = 1 / 3 * np.trace(epsilon)
```
Comment (psavery) [inline hexrd/ui/indexing/fit_grains_results_dialog.py:70]: @joelvbernier I'm double-checking the math. Is this right? hydrostatic strain is `1 / 3 * np.trace(epsilon)`, and hydrostatic stress is `1 / 3 * np.trace(sigma)`?

**Comment by cjh1** [comment]
@joelvbernier Can you double-check the math here?

**Comment by joelvbernier** [comment]
1/3 trace(<matrix in question>) is correct.

**Review by cjh1 — APPROVED** [review]

File: hexrd/ui/indexing/fit_grains_results_dialog.py (hunk 1 above)
Comment (psavery) [reply hexrd/ui/indexing/fit_grains_results_dialog.py:70]: Resolved by https://github.com/HEXRD/hexrdgui/pull/690#issuecomment-755519235

**Pull request merged**